glossary.rst

```diff
@@ -0,0 +1,67 @@
+Glossary
+========
+
+.. glossary::
+
+    Application
+        Website or application tracked by PPMS.
+
+    App ID
+        PPMS :term:`application` identificator (previously **website ID**, **site ID** or **idSite**).
+
+    User
+        Visitor on tracked :term:`application`.
+
+    Analytics ID
+        ID assigned to :term:`user` by :term:`Analytics` for the duration of :term:`Analytics` session. It's stored in
+        browser cookie.
+
+    User ID
+        Permanent ID assigned to :term:`user` by :term:`application` (e.g. username). You can read more about it
+        `here <https://help.piwik.pro/tag-manager/userid/>`_.
+
+    Device ID
+        Device ID (device identification) is a distinctive number associated with a smartphone or similar handheld
+        device. Device IDs are separate from hardware serial numbers.
+
+    Identifier
+        Unique :term:`user` ID (e.g. :term:`analytics ID`, :term:`user ID`, :term:`device ID` or email).
+
+    Visit
+        Period of continuous :term:`user` activity on :term:`application`. Its default maximal length is 30 minutes.
```

Comment:
This is not true. There is no such thing as max length of a visit.

A visit ends :
* after a period of inactivity (default 30 minutes)
* at midnight (enabled by default, see https://github.com/matomo-org/matomo/issues/6217 and follow up)
* when campaign changes (enabled by default, see https://matomo.org/faq/how-to/faq_19616/)